content/docs/en/getting-started/1-quick-start.md

```diff
@@ -9,19 +9,22 @@ it in the [NativeScript Playground](https://play.nativescript.org?template=play-
 If you are ready to move on and work locally on your machine, and [you
 have your system ready for native development](/en/docs/getting-started/installation), you can start by using a template:
 
-```shell
-$ tns create my-app --template nativescript-vue-template
-$ cd my-app
+###  Scaffold project
+```sh
```

Comment:
keep it `shell` for proper syntax highlighting

Reply:
done